tox.ini

```diff
@@ -362,7 +362,7 @@ deps =
     black==21.6b0
     click==8.0.2
     isort==5.9.3
-commands = aea generate-all-protocols --no-bump --check-clean
+commands = aea generate-all-protocols --check-clean
```

Comment:
`--no-bump` was causing the command to fail, I guess this was caused by https://github.com/valory-xyz/open-aea/pull/243

Reply:
Should be reverted after merging?